src/commands/prepare.ts

```diff
@@ -384,6 +393,14 @@ async function prepareChangelog(
       }
       if (!changeset.body) {
         replaceSection = changeset.name;
+        const changes = await getChangesSince(git, oldVersion);
+        changeset.body = changes
+          .map(change =>
+            change.pr
+              ? `- ${change.message}`
+              : `- ${change.message} (${change.hash.slice(0, 8)})`
+          )
+          .join('\n');
```

Comment:
So previous behavior was to leave `changeset.body` empty, and the new behavior is to populate it with the commit message. If there are no commit messages for the commits in question then `changelog.body` will remain empty.

How are newlines in commit messages handled? Gracefully?

Reply:
>So previous behavior was to leave changeset.body empty,

Not really, we filled it with the dreaded "No documented changes" text.

> the new behavior is to populate it with the commit message.
> How are newlines in commit messages handled? Gracefully?

Sort of. We only use the title, the first line of the commit message so no need to handle new lines. `simple-git` is a bit weird with its jargon: it uses `message` for the title and `body` for the actual long message.

>If there are no commit messages for the commits in question then changelog.body will remain empty

I think this is an interesting edge case and I think should be handled. Will follow up.

Reply:
>I think this is an interesting edge case and I think should be handled. Will follow up.

Okay, false alarm. Just checked the code and when we get an empty commit list, that results in a blank text which then triggers the default text fallback which still says "No documented changes.".